tests/UnifiedSpecTests/Operation.php

```diff
@@ -235,6 +235,10 @@ private function executeForChangeStream(ChangeStream $changeStream)
 
     private function executeForClient(Client $client)
     {
+        if ($this->name === 'clientBulkWrite') {
+            Assert::markTestSkipped('clientBulkWrite operation is not implemented');
+        }
```

Comment:
I was going to ask why you implemented the check here, but then realized that `Util::assertArgumentsBySchema()` asserts that that the operation is supported. That actually makes the `switch` statement's default case unreachable; although I'm not sure if tooling is smart enough to not complain about that.

I do think we should revisit how we track skipped tests, as splitting logic between the Operation class and test runner will ultimately make things harder to track. I've long wanted to support regex patterns to avoid enumerating individual test files and cases. This kind of feature detection also makes sense, although it would ideally be consolidated in a single place. Opened [PHPLIB-1540](https://jira.mongodb.org/browse/PHPLIB-1540) to track that idea.